src/main/java/no/nav/veilarbvedtaksstotte/service/KafkaConsumerService.java

```diff
@@ -58,6 +65,20 @@ public void behandleEndringPaOppfolgingsbruker(ConsumerRecord<String, KafkaOppfo
         }
     }
 
+    public void behandleEndringPaOppfolgingsbruker(ConsumerRecord<String, KafkaOppfolgingsbrukerEndringV2> kafkaOppfolgingsbrukerEndring) {
+        String fnr = kafkaOppfolgingsbrukerEndring.value().getFodselsnummer();
+        AktorId aktorId = aktorOppslagClient.hentAktorId(Fnr.ofValidFnr(fnr));
+        String oppfolgingsenhetId = kafkaOppfolgingsbrukerEndring.value().getOppfolgingsenhet();
+
+        Vedtak utkast = vedtaksstotteRepository.hentUtkast(aktorId.toString());
+
+        if (utkast != null && !utkast.getOppfolgingsenhetId().equals(oppfolgingsenhetId)) {
+            Enhet enhet = norg2Client.hentEnhet(oppfolgingsenhetId);
+            vedtaksstotteRepository.oppdaterUtkastEnhet(utkast.getId(), oppfolgingsenhetId);
+            beslutteroversiktRepository.oppdaterBrukerEnhet(utkast.getId(), oppfolgingsenhetId, enhet.getNavn());
+        }
+    }
+
```

Comment:
@matiasvinjevoll Er det slik at det "eneste" som skjer her er at dersom en bruker har flyttet enhet, vil utkastet bli lagt inn for brukeren på den nye enheten? Kunne funksjonen hett behandleFlyttingAvBruker eller lignende eller er det noe mer som skjer?

Reply:
Det stemmer, så hvis det skulle være meldinger om flere flyttinger for samme bruker, så vil tilstanden være feil i en kort periode, til siste melding for bruker er lest. Så tilstanden blir riktig når alle meldinger er lest.
Navnet kunne med fordel vært endret ja ➕  Ser ut til å være navngitt etter topic, i stede for hva man faktisk gjør